runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/EngineContext.java

```diff
@@ -105,6 +113,9 @@ String supplyNamespace(
     String supplyLocalName(
         long bindingId);
 
+    String supplyLocalName(
+        int localId);
+
```

Comment:
Do we actually need both forms of `supplyLocalName`?
Also, let's rename the `supplyNamespace` parameter from `bindingId` to `namespacedId` as it is general for all namespaced ids, not just bindings.

Reply:
done